modules/rosa-upgrading-manual-ocm.adoc

```diff
@@ -35,6 +35,12 @@ endif::sts[]
 For information about administrator acknowledgment, see xref:./../upgrading/rosa-upgrading-cluster-prepare.adoc#upgrade-49-acknowledgement_rosa-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
 . In the *Node draining* pane, select a grace period interval from the list. The grace period enables the nodes to gracefully drain before forcing the pod eviction. The default is *1 hour*.
 . In the *Update strategy* pane, click *Save* to apply your update strategy.
++
```

Comment:
@smunje1 Hi, The new note would be more catchy if we could move it below the node draining step(after line 36). WDYT?
Other sections LGTM. Thanks

Reply:
Yes. That's a great suggestion. Thanks!